index.html

```diff
@@ -3,9 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <title>Title</title>
+    <link rel="stylesheet" href="./styles/styles.css" />
   </head>
   <body>
-    <h1>Моя первая страница</h1>
-    <a href="./about/index.html">Об авторе</a>
+    <h1 class="heading">What do we offer</h1>
```

Comment:
You need to wrap you block in container (it can be

</div) and add to this container class with propperties:
max-width:1440px
margin: 0 auto
It is allow you to align your block to the center